modules/nw-ovn-kubernetes-migration-about.adoc

```diff
@@ -25,16 +25,21 @@ The subnets assigned to nodes and the IP addresses assigned to individual pods a
 
 While the OVN-Kubernetes network provider implements many of the capabilities present in the OpenShift SDN network provider, the configuration is not the same.
 
-* If your cluster uses any of the following OpenShift SDN capabilities, you must manually configure the same capability in OVN-Kubernetes:
+* If your cluster uses any of the following OpenShift SDN capabilities, the migration to OVN-Kubernetes can optionally migrate the configuration for these features as well:
 +
 --
-* Namespace isolation
 * Egress IP addresses
 * Egress network policies
```

Comment:
I think we'd better use 'Egress IP' and 'Egress Firewall' instead of 'Egress IP addresses' and 'Egress network policies' to refer to the name of the feature.